app/src/main/java/com/github/codetanzania/model/Service.java

```diff
@@ -28,6 +30,24 @@ public Service(String code, String name, String color) {
         this.color = color;
     }
 
+    public int getIcon() {
```

Comment:
To allow source code reuse, this function should be moved to a utility class e.g `public class ResourceUtils`. Also, rather than using issues' initials as test cases in a `switch` construct, the check should use something more versatile as open 311 service code.